templates/1_set_up_toffy.ipynb

```diff
@@ -15,9 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<p>Can't the sensitivity curve generation go in 4b, since it's only used there?  Is the idea that putting this here will encourage people to run a sweep before data aq?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>No, it's because it only needs to happen once per instrument, not separately for each run. So having it here means it won't be present in the notebook each time people are normalizing</p>